src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Debugging/LSPProximityExpressionsProvider .cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
There appears to be an errant ' ' in the name of this file.

Reply:
Woa, how'd that happen!